content/datasets/biblio_us17.md

```diff
@@ -0,0 +1,70 @@
+| **Year of Collection**   | 2017                                                                                                                                             |
+| **Attack Categories**    | Unknown                                                                                                                                          |
+| **User Emulation**       | Real users                                                                                                                                       |
+|                          |                                                                                                                                                  |
+| **Packed Size**          | 1,1 GB                                                                                                                                           |
+| **Unpacked Size**        | 6 GB                                                                                                                                             |
+| **Download Link**        | [must be requested](https://idus.us.es/items/53034f60-d215-44b0-a198-eb8a5229218e/request-a-copy?bitstream=b1b0c077-6f87-4160-8659-9dc34e269e3d) |
+
+***
+
+### Overview
+The Biblio-US17 dataset consists of selected features extracted from ~48 million web requests recorded from a webserver at the University of Seville (Spain).
+The recording period spanned 6,5 months and includes benign usage during that time.
+Requests are made available in a labeled, but heavily anonymized form.
+
+### Environment
+The web server in question is an Apache Web Server v2.2, traffic is scanned by a number of intrusion detection systems (Snort, Nemesida, Modsecurity with paranoia level 1 and 2).
+Further details, other than that this server is used in/by a library, are not available.
+
+### Activity
+Details, such as the purpose of this server within its environment, are not available.
+Data is recorded from 2017-01-01 to 2017-07-17, 2017, for a total of 198 days.
+
+### Contained Data
+Requests are grouped by day and each assigned an identifier of the form `[MM-DD-Fxxxxxx]`, with the first four digits representing the corresponding month and day, `F` signifying the protocol (A for HTTP, S for HTTPS) and the remainder being a unique number for that day.
+For each request, only the following information is available:
+- Method
+- URI (anonymized)
+- Protocol
+- Response code
+- Response size
+
+With an example looking like this:
+```
+[02-18-A001234] GET /2003/padron.html HTTP/1.1″ 200 11800
+```
+Notably, fine-grained timestamps are *not* available.
+Labels are available in a separate file;
+for each request a line beginning with the same identifier indicates which IDSs triggered on this request.
+Furthermore, additional labels inform about features like the confidence level of this attack which range from level 1 to 4, with level 1 being a confirmed attack.
```

Comment:
Are these really labeled by humans or are these simply e.g. the Suricata priorities?

Reply:
> Ground truth analysis entails attack detection via signature-based IDS systems and subsequent assisted review, with identified attacks manually categorized as true positives (TP) or false positives (FP). Furthermore, attacks are classified with four confidence/semantic levels:  [....]
I think this implies a manual process for these labels